src/components/views/elements/EventTilePreview.tsx

```diff
@@ -39,6 +39,7 @@ interface IProps {
     className: string;
 }
 
+/* eslint-disable camelcase */
```

Comment:
needs a re-enable? or a comment explaining why we don't have a re-enable at least.

Reply:
There is a re-enable at the bottom.